hydra-data/src/main/java/com/addthis/hydra/data/tree/concurrent/ConcurrentTreeNode.java

```diff
@@ -375,7 +375,7 @@ public boolean aliasTo(DataTreeNode node) {
         return true;
     }
 
-    HashMap<String, TreeNodeData> createMap() {
+    public HashMap<String, TreeNodeData> createMap() {
```

Comment:
let's push this visibility change down to virtual tree node. data has a complicated enough life cycle as it is, and it already causes contention where it shouldn't be required so it is on my short list for fixes if we start to have perf problems related to that.

don't want to risk it becoming even messier if we don't have to.